packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -120,6 +120,68 @@ lambdaAction.outputArtifact('Out2'); // returns the named output Artifact, or th
 See [the AWS documentation](https://docs.aws.amazon.com/codepipeline/latest/userguide/actions-invoke-lambda-function.html)
 on how to write a Lambda function invoked from CodePipeline.
 
+### Lambda deployed through CodePipeline
+
+If you want to deploy your Lambda through CodePipeline,
```

Comment:
Lambda deployed *exclusively* through CodePipeline, right? Might be worth a note that if you're using assets, you can override them anyway so no need to change your code then. I'm saying this because I want this to be the 99% case:

* Use an asset, use "cdk deploy" for "personal dev stack"
* That same asset is reused for CI/CD deployments.

I don't want the docs to send people down paths where they're going to be writing complicated conditionals because they think this is what they have to do to deploy via a Pipeline. Or worse, get confused and come ask us.

Maybe it's worth combining those 2 PRs so we can get the messaging and the examples right in one place?

Reply:
Since the other PR with overriding Assets has been closed without merging, I kept this title as-is. Let me know if that's fine, or if you still want to change it.